src/main/java/frc/robot/Robot.java

```diff
@@ -165,18 +166,37 @@ public void testInit() {
         CommandScheduler.getInstance().cancelAll();
     }
 
-    public ParallelCommandGroup singulatorAndIntake() {
+    public CommandBase singulatorAndIntake() {
+        CommandBase cmd = new ParallelCommandGroup(intake.intake(), indexer.indexMotor(.85));
+        cmd.setName("Intake Balls");
+        return cmd;
+    }
 
-        return new ParallelCommandGroup(intake.intake(), indexer.indexMotor(.85));
+    public CommandBase regurgitateFC() {
+        CommandBase cmd = new ParallelCommandGroup(intake.discharge(), indexer.reversePush());
+        cmd.setName("Regurgitate FC");
+        return cmd;
     }
 
-    public ParallelCommandGroup cancelCommand() {
+    public CommandBase passLine() {
+        CommandBase cmd = new SequentialCommandGroup(drive.driveDistance(40, .5));
```

Comment:
Why is this a group?

Reply:
I will be adding a shooting function to it soon once Austin finishes with his code

Reply:
👍